saleor/order/models.py

```diff
@@ -458,6 +459,15 @@ def can_edit(self):
     def get_total_quantity(self):
         return sum([line.quantity for line in self])
 
+    @property
+    def is_tracking_number_url(self):
+        validate = URLValidator()
+        try:
+            validate(self.tracking_number)
+            return True
+        except ValidationError:
+            return False
```

Comment:
We don't really care that much at this point whether the URL is RFC or not. Just check whether it is `^[-\w]+://`

Reply:
Great point. I'll change it to simple regex match in a moment